<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArchiveItemWidget</class>
 <widget class="QWidget" name="ArchiveItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>30</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="iconLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/resources/file-2x.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TextLabel" name="nameLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Archive_name</string>
     </property>
     <property name="elide" stdset="0">
      <string>Qt::ElideRight</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="detaiLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: grey;
}</string>
     </property>
     <property name="text">
      <string>281 MB  21.12.2014 18:52</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Preferred</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="restoreButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/action-undo-2x.png</normaloff>:/resources/action-undo-2x.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="inspectButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/info-2x.png</normaloff>:/resources/info-2x.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="deleteButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/resources/trash-2x.png</normaloff>:/resources/trash-2x.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/trash-2x.png</normaloff>:/resources/trash-2x.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete archive(s)</string>
   </property>
  </action>
  <action name="actionInspect">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/info-2x.png</normaloff>:/resources/info-2x.png</iconset>
   </property>
   <property name="text">
    <string>Inspect</string>
   </property>
   <property name="toolTip">
    <string>Inspect archive</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/action-undo-2x.png</normaloff>:/resources/action-undo-2x.png</iconset>
   </property>
   <property name="text">
    <string>Restore</string>
   </property>
   <property name="toolTip">
    <string>Restore Archive</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextLabel</class>
   <extends>QLabel</extends>
   <header location="global">textlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
